foundryup/install

```diff
@@ -48,9 +64,11 @@ if [[ ":$PATH:" != *":${FOUNDRY_BIN_DIR}:"* ]]; then
 fi
 
 # Warn MacOS users that they may need to manually install libusb via Homebrew:
-if [[ "$OSTYPE" =~ ^darwin && ! -f /usr/local/opt/libusb/lib/libusb-1.0.0.dylib ]]; then
```

Comment:
We need to check both. M1s use the `/opt` path, intel macs use `/usr/local`

Reply:
> We need to check both. M1s use the `/opt` path, intel macs use `/usr/local`

ACK

tho this also is using `.bashc`, not `.bash_profile` which is the correct source for user info if your using bash on macOS.

Reply:
[`a94a0c2` (#3907)](https://github.com/foundry-rs/foundry/pull/3907/commits/a94a0c29ca01a9a3a49d1b6f7d0bea4a7ab6e3ba#diff-79a844de4d6869f243ecd1e7ca4d9532d9a4987f85ca63d9f4af85aa227ecb38R68-R101)

I refactored this - I think this approach more robust, LMK ser

Reply:
> We need to check both. M1s use the `/opt` path, intel macs use `/usr/local`

@brockelmore think this is good